src/MarginCalculator.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace PivotLibre\Tideman;
+
+use PivotLibre\Tideman\MarginRegistry;
+
+class MarginCalculator
```

Comment:
'MarginCalculator' sounds kind of generic. Should we call it a 'MarginTallier'  instead?